raiden/channel.py

```diff
@@ -246,6 +246,7 @@ def compute_proof_for_lock(self, secret, lock):
         lock_encoded = bytes(lock.as_bytes)
         lock_hash = sha3(lock_encoded)
         merkle_proof = [lock_hash]
+        # Why is the return value not used here?
```

Comment:
Merkleroot is a destructive function that changes the `merkle_proof` in place, so the function is being executed just for the side effects.